asmcomp/arm64/proc.ml

```diff
@@ -270,15 +270,15 @@ let destroyed_at_reloadretaddr = [| |]
 
 let safe_register_pressure = function
   | Iextcall _ -> 8
-  | Ialloc _ -> 24
-  | _ -> 25
+  | Ialloc _ -> 22
+  | _ -> 23
```

Comment:
22 is because `Ialloc` uses register `r8` as temporary, decreasing by one the number of integer registers that can be allocated to OCaml values across an `Ialloc` instruction.

You're right about the callee-save registers: 8 is wrong, I'll fix ASAP.